PR #87 breaks the use of MinGW make on Windows

https://github.com/nvim-telescope/telescope-fzf-native.nvim/pull/87

Output follows. Note that I have to type `exit` to exit the interactive shell.

```
C:\Users\bakkenl\AppData\Local\nvim\pack\nvim-telescope\start\telescope-fzf-native.nvim>make --version
GNU Make 4.3
Built for Windows32
Copyright (C) 1988-2020 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

C:\Users\bakkenl\AppData\Local\nvim\pack\nvim-telescope\start\telescope-fzf-native.nvim>git clean -xffd

C:\Users\bakkenl\AppData\Local\nvim\pack\nvim-telescope\start\telescope-fzf-native.nvim>make
cmd //C mkdir build
Microsoft Windows [Version 10.0.22000.1219]
(c) Microsoft Corporation. All rights reserved.

C:\Users\bakkenl\AppData\Local\nvim\pack\nvim-telescope\start\telescope-fzf-native.nvim>exit
gcc -O3 -Wall -Werror -fpic -std=gnu99 -shared src/fzf.c -o build/libfzf.dll
c:/programdata/chocolatey/lib/mingw/tools/install/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/12.2.0/../../../../x86_64-w64-mingw32/bin/ld.exe: cannot open output file build/libfzf.dll: No such file or directory
collect2.exe: error: ld returned 1 exit status
make: *** [Makefile:18: build/libfzf.dll] Error 1
```

The fix is to revert the extra escaping introduced in #87, or to improve system detection.

I'll open a PR to address this.

--- a/Makefile
+++ b/Makefile
@@ -1,10 +1,16 @@
 CFLAGS = -Wall -Werror -fpic -std=gnu99
 
 ifeq ($(OS),Windows_NT)
-    MKD = cmd //C mkdir
-    RM = cmd //C rmdir //Q //S
     CC = gcc
     TARGET := libfzf.dll
+ifeq (,$(findstring MSYS,$(MSYSTEM)))
+	# On Windows, but NOT msys
+    MKD = cmd /C mkdir
+    RM = cmd /C rmdir /Q /S
+else
+    MKD = mkdir -p
+    RM = rm -rf
+endif
 else
     MKD = mkdir -p
     RM = rm -rf

